example_sns_init.yaml

```diff
@@ -25,104 +31,307 @@ description: >
+    # decentralization swap to succeed. This guarantees that a newly created SNS
+    # will have a minimum number of unique users (decentralization factor).
+    # If a decentralization swap's maximum target is reached, or the
+    # deadline elapses and there have not been at least `minimum_participants`,
+    # the swap will fail.
     minimum_participants: 57
 
-    minimum_icp:   100_000 tokens
+    # The total number of ICP that is required for a decentralization swap to
+    # succeed. This number divided by the number of SNS tokens being offered
+    # gives the reserve price of the swap, i.e., the minimum number of ICP per
+    # SNS token. If this amount is not achieved, the swap will fail.
+    minimum_icp: 100_000 tokens
+
+    # The maximum number of ICP that is targeted by this decentralization swap.
+    # If this amount is achieved with sufficient participation, the swap will
+    # succeed immediately, without waiting for the deadline. This means that
+    # a participant knows the minimum number of SNS tokens received per invested
+    # ICP. If this amount is achieved without reaching `minimum_participants`,
+    # the swap will immediately fail without waiting for the due date.
     maximum_icp: 1_000_000 tokens
 
+    # The minimum amount of ICP that each participant must contribute
+    # to participate.
     minimum_participant_icp:     10 tokens
+
+    # The maximum amount of ICP that each participant must contribute
+    # to participate.
     maximum_participant_icp: 10_000 tokens
 
+    # The optional text that swap participants must confirm before they may
+    # participate in the swap. If set, must be within 1 to 1,000 characters.
     # confirmation_text: >
-    #     Do you understand the responsibilities and risks associated
-    #     with participating in this token swap?
+    #     I confirm my understanding of the responsibilities and risks
+    #     associated with participating in this token swap.
 
+    # An optional list of countries that should not participate in the
+    # decentralization swap. By default, participants from all countries
+    # are allowed.
     restricted_countries:
         - AQ  # Antarctica
 
-    # Declare that each participant will get three SNS neurons in the following
-    # states:
-    # 1. Unlocked
-    # 2. Locked for 1 month
-    # 3. Locked for 2 months
+    # Configuration of the vesting schedule of the basket of Neurons each
+    # participant will receive from a successful decentralization swap.
     VestingSchedule:
+        # The number of events in the vesting schedule. This translates to how
+        # many neurons will be in each participant's neuron basket.
         events: 3
+
+        # The interval at which the schedule will be increased per event. The
+        # first neuron in the basket will be unlocked with zero dissolve delay.
+        # Each other neuron in the schedule will have its dissolve delay
+        # increased by `interval`.
```

Comment:
What is the vesting period of the SNS neurons?

Reply:
IIUC the neurons in the basket have no vesting_period, only the developer neurons have.
So this field is just not set. 
I see that it might be a bit confusing that we talk about vesting here but don't use this field that has vesting in the name. Maybe we can just add an explicit comment about this?

Reply:
Added:

> Note that the notion of `Distribution.neurons.vesting_period` applies only to developer neurons. While neuron basket neurons do not use `vesting_period`, they have a vesting schedule.